internal/config/config.go

```diff
@@ -101,14 +105,23 @@ func Connect(cfg DbCfg, tableName string) *sql.DB {
 }
 
 func InitSvcConfig(cfg Config) *SvcConfig {
-	// init required services and assign to the service struct fields
 	dataBase := Connect(cfg.DataBase, cfg.DataBase.TableName)
 	jwtSvc := jwtSvc.JWTAuthService(cfg.SecretKey)
 	msgBrokerSvc := sdk.NewMsgBrokerSvc(cfg.MessageQueue.SvcUrl)
 	id, err := msgBrokerSvc.RegisterPub(cfg.MessageQueue.NewAccountChannel)
 	if err != nil {
 		panic(err.Error())
 	}
+	privateKey, err := crypt.PEMStrAsPrivKey(cfg.MessageQueue.Key)
+	if err != nil {
+		panic(err.Error())
+	}
+	publicKey := privateKey.PublicKey
+	pubKey := crypt.PubKeyAsPEMStr(&publicKey)
+	err = msgBrokerSvc.RegisterSub("PUT", "http://localhost/v1/activate", pubKey, cfg.MessageQueue.ActivatedAccountChannel)
```

Comment:
What happens if the service is started on a port other than 80 or when the service is hidden under a DNS(let's say via docker compose)? The base service url should be determined from the `config.json` to that it is in sync with the actual `host:port` of the service server.